packages/flutter_tools/lib/src/ios/devices.dart

```diff
@@ -413,11 +413,32 @@ class IOSDevice extends Device {
       }
 
       _logger.printTrace('Application launched on the device. Waiting for observatory url.');
-      final Timer timer = Timer(discoveryTimeout ?? const Duration(seconds: 30), () {
-        _logger.printError('iOS Observatory not discovered after 30 seconds. This is taking much longer than expected...');
+      final int defaultTimeout = interfaceType == IOSDeviceConnectionInterface.network ? 60 : 30;
```

Comment:
I found that this took longer when using a network device. 60 seconds might be a bit generous, though. Sometimes 30 seconds is enough, but sometimes it's more like 35. Not sure if people's internet connections might influence this (mine is pretty fast), which is why I overestimated.

Reply:
Wow that's a long time.  Maybe 45 and we'll see if we get complaints?